sdk/communication/Azure.Communication.Common/tests/Pipeline/HMACAuthenticationPolicyTests.cs

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+using NUnit.Framework.Internal;
+
+namespace Azure.Communication.Pipeline
+{
+    public class HMACAuthenticationPolicyTests : PolicyTestBase
+    {
+        private const string secret_key = "68810419818922fb0263dd6ee4b9c56537dbad914aa7324a119fce26778a286e";
```

Comment:
nit: naming -> SecretKey I was surprised to see _ in the name of the original code. We do not normally use it in C#